app/src/renderer/js/preferences.js

```diff
@@ -208,7 +226,7 @@ document.addEventListener('DOMContentLoaded', async () => {
   };
 
   fpsSlider.onchange = function () {
-    app.kap.settings.set('fps', this.value);
+    app.kap.settings.set('fps', Number(this.value));
```

Comment:
For some good news, the Aperture upgrade* helped catch this subtle bug.

*The Codable stuff: https://github.com/wulkano/aperture/commit/0193e72e2a78ceaeae5178cde852686113c6da59